container/test-app/index.html

```diff
@@ -417,6 +513,36 @@
 
         //...
       });
+      let readyTest = document.getElementById('readyTest');
+      if (!readyTest.isMfeInitialized) {
+        readyTest.addEventListener(MFEventID.INITIALIZED, () => {
+          console.log('mfe is ready now!');
+        });
+      } else {
+        console.log('mfe immediately ready');
+      }
+      let readyTest2 = document.getElementById('readyTest2');
+      if (!readyTest2.isMfeInitialized) {
+        readyTest2.addEventListener(MFEventID.INITIALIZED, () => {
+          console.log('mfe is ready now! 2');
+        });
+      } else {
+        console.log('mfe immediately ready 2');
+      }
+      let readyTest3 = document.getElementById('calendar');
+      if (!readyTest3.isMfeInitialized) {
+        readyTest3.addEventListener(MFEventID.INITIALIZED, () => {
+          console.log('calendar is ready now! 3');
+        });
+      } else {
+        console.log('calendar immediately ready 3');
+      }
+      let dashboard2 = document.getElementById('dashboard2');
```

Comment:
This is getting very confusing. Could you change the id name to something more specific. 
I believe we will have to use them when we start making e2e-tests for container.

readyTest -> test-defer-init-false
readyTest2 -> test-defer-init-true
dashboard2 -> nested-wc-compound-container